liveness-detector/Dockerfile

```
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: 2024 Idiap Research Institute <contact@idiap.ch>
+# SPDX-FileContributor: Salim Kayal <salim.kayal@idiap.ch>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ghcr.io/prefix-dev/pixi:0.24.2-bullseye-slim AS build
```

suggestion: Lets use bookworm which is the latest stable version of debian.

```suggestion
FROM ghcr.io/prefix-dev/pixi:0.24.2-bookworm-slim AS build
```

I checked and they have the bookworm image.